spantest/inmemory.go

```diff
@@ -77,7 +77,31 @@ func (fx *InMemoryFixture) NewDatabaseFromStatements(t *testing.T, statements []
 	for i, statement := range statements {
 		ddl, err := spansql.ParseDDL(fmt.Sprintf("statement%d", i), statement)
 		assert.NilError(t, err)
+		washDDL(ddl)
 		assert.NilError(t, server.UpdateDDL(ddl))
 	}
 	return client
 }
+
+// washDDL removes any statements which are not yet supported by
+// the spannertest package for the purpose of not preventing new spanner
+// features from breaking existing tests
+func washDDL(ddl *spansql.DDL) {
```

Comment:
let's call it `removeUnsupportedStatements` to make it a bit more explicit?

Reply:
Fixed